torch/_subclasses/meta_utils.py

```diff
@@ -265,6 +266,49 @@ def sym_sizes_strides_storage_offset(t):
                         with torch.enable_grad():
                             r = r.clone()
                             r._coalesced_(t.is_coalesced())
+                elif t.is_nested:
```

Comment:
start of the logic for converting real NT -> meta / fake instance of NestedTensor python subclass